webapp/app/components/roomList/roomList.component.js

```diff
@@ -25,6 +27,12 @@ export class RoomListComponent extends PureComponent {
     rooms: Map(),
   };
 
+  isRoomActive = (id) => ifElse(
```

Comment:
arrow function niepotrzebne w sumie

Reply:
No właśnie chyba potrzebne bo inaczej masz inny kontekst funkcji i nie masz dostępu do propsów przez this

Reply:
z tego co pamietam isRoomActive bedzie automarycznie zbindowane do this, wiec bez problemu dostaniesz sie do this.props